### PR TITLE
Develop appimporter

### PR DESCRIPTION
**Pull request by fredericcervenansky**
Redmine feature 2909: challenge MICCAI template

### CODE REVIEW

File: Portal/vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/Constants.java [hunk 1]
```diff
@@ -72,4 +72,13 @@
...
     //vm files
     public static final String VM_GASW = "gasw.vm";
+    //LFN access to SegPerfAnalyzer
+    public static final String LNF_PATH = "";
+     // Application Importer Challenge Descriptor Path
+     public static final String APP_IMPORTER_CHALLENGE_PATH_MSSEG = "/vip/MSEG-admin/applications/templates" ;
+     public static final String APP_IMPORTER_CHALLENGE_PATH_PETSEG = "/vip/PETSEG-admin/applications/templates" ;
+     // Application Importer Challenge Descriptor
+     public static final String APP_IMPORTER_CHALLENGE_METRIC = "SegPerfAnalyzer.json" ;
+     public static final String APP_IMPORTER_CHALLENGE_METADATA = "metadata-updater.json" ;
```
Comment (glatard) [inline Portal/vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/Constants.java]: Weird indentation (leading space) in the lines above.

File: Portal/vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/DisplayTab.java [hunk 2]
```diff
@@ -102,19 +111,66 @@ private void configure() {
...
+                else
+                   createApplication(); 
+                
             }
         });
         createApplicationButton.setWidth(120);
         globalLayout.addMember(createApplicationButton);
     }
 
+    public void createApplicationWithAddDesc() {
```
Comment (glatard) [inline Portal/vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/DisplayTab.java]: What about adding a line of comment to explain what this method does?

File: Portal/vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/VIPLayout.java [hunk 3]
```diff
@@ -56,7 +76,42 @@ public VIPLayout(String width, String height) {
         
         overwriteIfexists = new CheckboxItem("ckbox_over","Overwrite application version if it exists");
         overwriteIfexists.setAlign(Alignment.LEFT);
-
+        
+          appCbItem = new SelectItem();
```
Comment (glatard) [inline Portal/vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/VIPLayout.java]: Indendation issue.

File: Portal/vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/VIPLayout.java [hunk 4]
```diff
@@ -56,7 +76,42 @@ public VIPLayout(String width, String height) {
...
+        appCbItem.setTitle("<b>Select type of application</b>");
+       // appCbItem.setHint("<nobr>select type of application</nobr>");
+        appCbItem.setType("comboBox");
+        LinkedHashMap<String, String> valueMap = new LinkedHashMap<String, String>();  
+        valueMap.put("none", "none");
```
Comment (glatard) [inline Portal/vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/VIPLayout.java]: Maybe "standalone" would be a better word than "none".

**Comment by fredericcervenansky** [comment]
Corrections based on Tristan comments

**Pull request merged**